Utils/Dataflow/055_documents2TTL/documents2ttl.py

```diff
@@ -399,86 +401,66 @@ def fix_list_values(list_vals):
         item = fix_string(item)
     return list_vals
 
-def write_ttl2file(output, ttl_string):
+def process(stage, msg):
     """
-    write ttl string with document metadata to TTL file
-    :param output:
-    :param ttl_string:
+    Processing messages from JSON to TTL
+    :param stage: instance of JSON2TTLProcessorStage
+    :param msg: input JSON message
     :return:
     """
-    try:
-        ttl_file = open(output, "w+")
-    except IOError:
-        sys.stderr.write('cannot open file')
-    else:
-        try:
-            ttl_file.write(ttl_string)
-            ttl_file.write('\n\0')
-        except IOError:
-            sys.stderr.write('can\'t write to file')
-        else:
-            ttl_file.close()
-            sys.stderr.write("TTL file has written!")
+    data = msg.content()
+    paper_id = data.get('dkbID')
+
+    # papers processing
+
+    doc_iri = get_document_iri(paper_id)
+    doc_ttl = ""
+    doc_ttl += '{docIRI} a <{ontology}#Paper> .\n' \
+        .format(docIRI=doc_iri, ontology=ONTOLOGY)
+    doc_ttl += document_glance(data.get('GLANCE'), doc_iri, PAPER_GLANCE_ATTRS)
+    doc_ttl += document_cds(data.get('CDS'), doc_iri, PAPER_CDS_ATTRS)
+
+    # supporting documents processing
+
+    if "supporting_notes" in data:
+        for note in data.get('supporting_notes'):
+            note_id = note.get('dkbID')
+            note_iri = get_document_iri(note_id)
+            doc_ttl += '{noteIRI} a <{ontology}#SupportingDocument> .\n' \
+                .format(noteIRI=note_iri, ontology=ONTOLOGY)
+            doc_ttl += document_glance(note.get('GLANCE'), note_iri, NOTE_GLANCE_ATTRS)
+            doc_ttl += document_cds(note.get('CDS'), note_iri, NOTE_CDS_ATTRS)
+
+    doc_ttl += documents_links(data)
+    ttl_message = messages.TTLMessage(doc_ttl)
+    output_message = pyDKB.dataflow.Message(pyDKB.dataflow.messageType.TTL)(ttl_message.content())
```

Comment:
`pyDKB.dataflow.Message(messageType)` is a function that returns a corresponding class `*Message` (here -- `TTLMessage`), so this can be avoided: `ttl_message` and `output_message` are completely the same.